app.js

```diff
@@ -1,6 +1,61 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const keys = require('./config/keys');
+require('./models/Property');
+require('./models/Owner');
+const Property = mongoose.model('property');
+
+mongoose.connect(keys.mongoUrl);
+
 const app = express();
 
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/api/create', (req, res) => {
+  const { name, price, location, p_type, imageUrl, size } = req.body;
+  console.log(req.body);
+  const property = new Property({
+    name,
+    price,
+    location,
+    p_type,
+    imageUrl,
+    size,
+  });
+
+  property
+    .save()
+    .then(house => {
+      res.send(house);
+    })
+    .catch(err => {
+      res.status(400).send('unable to save in the database');
+    });
+});
+
+app.get('/api/fetch', (req, res) => {
+  Property.find()
+    .then(properties => {
+      res.send(properties);
+    })
+    .catch(err => {
+      res.status(400).send('unable to read from the database');
+    });
+});
+
+app.get('/api/selected/:id', (req, res) => {
```

Comment:
also let's move those routes to a router file 
## modularisation at it's finest :confetti_ball:

Reply:
also not just having a different file for the routes but a different folder that's called `controllers` that contains the logic for each route, 
-- router.js
-- app.js
--controllers/
   ---- fetch.js
   ---- getSomething.js

Reply:
Done